MetaMask/metamask-extension
# Fixing Dapp link on NFT import screen

Fixes: https://github.com/MetaMask/metamask-extension/issues/19679

## Explanation
This fixes the link to the dapp that is present on the import screen when adding an NFT. The issue was that the URL host was being used for the link `href`. This adds a second var to keep track of the link, while still maintaining the host for display purposes. 

### Before
See video in linked issue

### After
https://github.com/MetaMask/metamask-extension/assets/8732757/acf590be-78b6-477b-a7ca-c017d5a1d744

## Manual Testing Steps
1. Navigate to Test dapp
2. Deploy an NFT contract
3. Mint an NFT
4. Go to watch the NFT
5. Ensure the link to the dapp clicks through as expected

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
